Libraries/libs.mk

```diff
@@ -13,7 +13,9 @@ LIBS_DIR ?= $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 # ************************
 LIB_BOARD ?= 1
 ifeq ($(LIB_BOARD), 1)
-BOARD_DIR := $(LIBS_DIR)/Boards/$(TARGET_UC)/$(BOARD)
+BSP_SEARCH_DIR ?= $(LIBS_DIR)/Boards/$(TARGET_UC)
+BOARD_DIR := $(BSP_SEARCH_DIR)/$(BOARD)
+PROJ_CFLAGS += -DLIB_BOARD
```

Comment:
As per of my above comment, 
I recommend you to delete -DLIB_BOARD define.

Reply:
https://github.com/Analog-Devices-MSDK/msdk/pull/613#issuecomment-1591866010